utils/ChromiumDownloader.js

```diff
@@ -25,7 +25,10 @@ const removeRecursive = require('rimraf');
 const ProxyAgent = require('https-proxy-agent');
 const getProxyForUrl = require('proxy-from-env').getProxyForUrl;
 
-const DOWNLOADS_FOLDER = path.join(__dirname, '..', '.local-chromium');
+const DOWNLOADS_FOLDER = process.env.npm_config_puppeteer_downloads_folder ||
```

Comment:
the variable `npm_config_puppeteer_download_folder` is defined only when script is run as part of npm step. This is not always the case for `ChromiumDownlaoder`: it is used both from `install.js` (npm script) and `lib/Launcher.js` (not an npm script).

Reply:
Yes. This is expected behavior that specifying `DOWNLOADS_FOLDER` in any time with `npm_config_`. But this change may spark more usage issues, so i will remove this specifying.